values.yaml

```diff
@@ -181,9 +222,17 @@ global:
       storageClassName:
         _default: hostpath
         local: hostpath
-        aws: efs-sc
+        aws: ebs-sc
       name: redis-persistence-slave
-      storage: 4Gi
+      auto:
+        _default: true
+        local: true
+        aws: true
+      mode:
+        _default: ReadWriteOnce
+        local: ReadWriteOnce
+        aws: ReadWriteOnce
```

Comment:
I presume the statefulset redis/slave/{n} mount path means this would only ever be ReadWriteOnce?

Reply:
Exactly.